Modules/Core/Common/src/itkPlatformMultiThreaderPosix.cxx

```diff
@@ -30,11 +30,7 @@
 #include "itkObjectFactory.h"
 #include "itksys/SystemTools.hxx"
 #include <unistd.h>
-
-#ifdef __APPLE__
-#  include <sys/types.h>
-#  include <sys/sysctl.h>
-#endif
+#include <pthread.h>
```

Comment:
`pthread.h` gets included in `itkThreadSupport.h`.

Reply:
Removed.